CHANGELOG.md

```diff
@@ -21,6 +21,9 @@ as necessary. Empty sections will not end in the release notes.
   `commits`, `references` and `entries` commands.
 - Helm charts: OpenTelemetry SDK is now completely disabled when tracing is disabled.
 - Helm charts: when auth is disabled, Quarkus OIDC doesn't print warnings anymore during startup.
+- `MultiEnvTestEngine`: this JUnit5 test engine used in multi-environment tests now reports the 
```

Comment:
Let's remove the changelog entry - this PR changes only test code, nothing user-facing.

Reply:
I agree. The PR will be in the detailed list of changes (at least in GH) anyway.